front/app/containers/IdeasShowPage/index.tsx

```diff
@@ -97,19 +90,22 @@ const IdeasShowPage = () => {
     return <PageNotFound />;
   }
 
-  if (idea) {
+  if (idea && project) {
+    const isIdeaInCurrentPhase =
+      idea.data.relationships.phases.data.filter(
+        (phase) => phase.id === participationContext?.id
+      ).length > 0;
```

Comment:
Reduced to 1 var instead of 2. I try to use only vars if they consist of code of which it's not easy to determine what it means, so the var name can speed up understanding or if used more than 1x. Otherwise, I delete them.